apps/explorer/lib/explorer/chain.ex

```diff
@@ -6768,6 +6768,20 @@ defmodule Explorer.Chain do
     end
   end
 
+  def is_validator_address_signer_address?(address_hash) do
+    signer_query =
+      from(validator in CeloValidator,
+        where: validator.signer_address_hash == ^address_hash
+      )
+
+    signer_query
+    |> Repo.one()
```

Comment:
I believe it would be more concise if you use `Repo.aggregate(where(CeloValidator, signer_address_hash: ^address_hash), :count, :address)` (havent' tested though). Then you can compare it to a number and get the boolean.